client/components/signup-form/social/style.scss

```diff
@@ -0,0 +1,27 @@
+/*rtl:ignore*/
```

Comment:
Per [these docs](https://github.com/Automattic/wp-calypso/blob/master/docs/components.md), I think the component for these social buttons should either be a standalone component (in `/client/components`) or a sub-component in the same directory as `SignupForm`. Then, instead of styles like `.signup-form__social__button` we would either have:

- `.social-buttons__button` if we created a standalone component named `SocialButtons`.
- `.signup-form__social-button` if we created a sub-component.